lib/ansible/modules/system/ufw.py

```diff
@@ -292,12 +297,16 @@ def ufw_version():
 
     # Ensure ufw is available
     ufw_bin = module.get_bin_path('ufw', True)
+    grep_bin = module.get_bin_path('grep', True)
 
     # Save the pre state and rules in order to recognize changes
-    (_, pre_state, _) = module.run_command(ufw_bin + ' status verbose')
-    (_, pre_rules, _) = module.run_command("grep '^### tuple' /lib/ufw/user.rules /lib/ufw/user6.rules /etc/ufw/user.rules /etc/ufw/user6.rules")
+    (_, pre_state, _) = execute([[ufw_bin], ['status'], ['verbose']])
+    (_, pre_rules, _) = get_current_rules()
 
     changed = False
+
+    debug = ""
```

Comment:
What do you need this for?

Reply:
Oops, I forget to remove that temporary var.